Header header
float64 score
float32[9] homography
float32 alpha
float32 beta
geometry_msgs/Point[4] corners

